platform/pulpcore/app/serializers/repository.py

```diff
@@ -202,9 +206,21 @@ class DistributionSerializer(ModelSerializer):
     )
     name = serializers.CharField(
         help_text=_('The name of the distribution. Ex, `rawhide` and `stable`.'),
+        validators=[validators.MaxLengthValidator(
+            models.Distribution._meta.get_field('name').max_length,
+            message=_('Distribution name length must be less than {} characters'.format(
```

Comment:
I thought `format` needs to be called on what's returned from `_()`.

Reply:
Yes the format needs to be called on the return string from gettext aka `_()`. Otherwise the string translation would include the specific filled variables which would require and endless amount of unpredictable string translations to exist since runtime can produce a huge number of those unique strings.